cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -102,16 +102,6 @@ public void run() {
                 editBox.setTextColor(Color.WHITE);
                 editBox.setSingleLine();
                 editBox.setOpenGLViewScaleX(scaleX);
-                Resources r = mCocos2dxActivity.getResources();
```

Comment:
@anjandaffo  
If the new default you changed is better than the old, I think it's OK.
The old way use some magic number and I really hate magic number. 
But if the new way break some back compatibility, especially for the visually looks, it will make our developers unhappy.